homeassistant/components/smarttub/entity.py

```diff
@@ -65,3 +65,24 @@ def __init__(
     def _state(self):
         """Retrieve the underlying state from the spa."""
         return getattr(self.spa_status, self._state_key)
+
+
+class SmartTubExternalSensorBase(SmartTubEntity):
+    """Class for additional BLE wireless sensors sold separately."""
+
+    def __init__(self, coordinator, spa, sensor: smarttub.SpaSensor) -> None:
+        """Initialize the external sensor entity."""
+        self.sensor_address = sensor.address
+        self._attr_unique_id = f"{spa.id}-externalsensor-{sensor.address}"
+        super().__init__(coordinator, spa, self._human_readable_name(sensor))
+
+    @staticmethod
+    def _human_readable_name(sensor):
```

Comment:
Idem

Reply:
I'm also wondering what the name could be, can someone attach 2 of these sensors?